ui-admin/src/study/kits/KitList.tsx

```diff
@@ -42,16 +103,59 @@ export default function KitList({ studyEnvContext }: { studyEnvContext: StudyEnv
     loadKits()
   }, [studyEnvContext.study.shortcode, studyEnvContext.currentEnv.environmentName])
 
+  const kitsByStatus = _groupBy(kits, kit => {
+    return statusTabs.find(tab => tab.status === kit.pepperStatus?.currentStatus)?.status || 'ERROR'
+  })
+
+  const tabLinkStyle = ({ isActive }: {isActive: boolean}) => ({
+    borderBottom: isActive ? '3px solid #333': '',
+    background: isActive ? '#ddd' : ''
+  })
+
   return <LoadingSpinner isLoading={isLoading}>
-    <KitListView kits={kits} studyEnvContext={studyEnvContext}/>
+    <div className="container">
+      <div className="d-flex w-100" style={{ backgroundColor: '#ccc' }}>
+        { statusTabs.map(tab => {
+          const kits = kitsByStatus[tab.status] || []
+          return <NavLink key={tab.label} to={tab.label.toLowerCase()} style={tabLinkStyle}>
+            <div className="py-3 px-5">
+              {kits?.length} {tab.label}
+            </div>
+          </NavLink>
+        })}
+      </div>
+      <Routes>
+        <Route index element={<Navigate to={statusTabs[0].label.toLowerCase()} replace={true}/>}/>
+        { statusTabs.map(tab => {
+          return <Route key={tab.label} path={tab.label.toLowerCase()} element={
```

Comment:
it feels backwards to base route paths off of labels.  If you don't want to create a separate "path" property for each tab, you could run the dependency in the other direction, and have the tabs have a "key" and then the label is the key title cased.